conda-recipe/run_test.sh

```diff
@@ -18,7 +18,18 @@
 # limitations under the License.
 #*******************************************************************************
 
-python -c "import daal4py"
-mpirun -n 4 python -m unittest discover -v -s tests -p spmd*.py
-pytest --pyargs daal4py/sklearn/
-python tests/run_tests.py
+ok=0
+
+if ! python -c "import daal4py"; then
```

Comment:
For check of last return code exist `$?`
```suggestion
python -c "import daal4py"
ok += $?
```

Reply:
What if exit code is not equal 0 or 1?